modules/apps/app-builder/app-builder-web/src/main/resources/META-INF/resources/js/pages/apps/edit/EditAppContext.es.js

```diff
@@ -160,6 +161,36 @@ const reducer = (state, action) => {
 
 			return uppdateAppDeployment(state, PRODUCT_MENU, newAppDeployment);
 		}
+		case UPDATE_WORKFLOW_PROCESS_ID: {
+			const {
+				active,
+				appDeployments,
+				dataLayoutId,
+				dataListViewId,
+				name,
+				scope,
+			} = state.app;
+
+			const app = {
+				active,
+				appDeployments,
+				dataLayoutId,
+				dataListViewId,
+				name,
+				scope,
+			};
+
+			if (action.id) {
+				app.workflowProcessId = action.id;
+			}
+
```

Comment:
Is it needed? 

Would that be enough? passing action.id as `undefined` to the option "No Workflow"?

`return { ...state, app: {...state.app, workflowProcessId: action.id } }`

Reply:
The app creation API has no support for workflowProcessId variable even we sending it undefined, it will return an error at the moment of app creation